tcpdf.php

```diff
@@ -5467,6 +5467,9 @@ protected function getCellCode($w, $h=0, $txt='', $border=0, $ln=0, $align='', $
 				$s .= 'q '.$this->TextColor.' ';
 			}
 			// rendering mode
+			if(!is_numeric($this->textstrokewidth)){
+ 				$this->textstrokewidth = 0;
+			}
```

Comment:
Can you try to track in the code where it changes to a non numeric value ?

```php
	/**
	 * Text stroke width in doc units.
	 * @protected
	 * @since 4.9.008 (2010-04-03)
	 */
	protected $textstrokewidth = 0;
```
Since it's initialized to 0
Maybe you can add an `@var int` or `@var float` or `@var int|float` phpdoc on this property, for documentation purposes


The fix you propose indeed should work, but if we can catch it earlier to avoid the `textstrokewidth` being a non numeric that would be cool

Reply:
OK, turns out I was passing a non-numeric value to the `Text` function for `$fstroke`. The new commit ignores non-numeric values that are passed

Reply:
Thanks, it looks way better

Is this mistake related to your code of tcpdf code calling the function ?

Reply:
I guess so, but anybody calling `$pdf->Text($x, $y, $text, "non-numeric");` would get the same warning